build/common.sh

```diff
@@ -862,7 +862,13 @@ function kube::release::create_docker_images_for_server() {
 
         rm -rf ${docker_build_path}
         mkdir -p ${docker_build_path}
-        ln ${binary_dir}/${binary_name} ${docker_build_path}/${binary_name}
+        ln ${binary_dir}/${binary_name} ${docker_build_path}/${binary_name} && rc=$? || rc=$?
+        if [ $rc != 0 ]; then
```

Comment:
Why is this not just..
```
if ! ln ${binary_dir}/${binary_name} ${docker_build_path}/${binary_name}; then
```
instead of jumping through hoops? You're not doing anythign with the `rc` later.